tests/selenium/start.js

```diff
@@ -5,64 +5,16 @@ var driver = new webdriver.Builder().
   withCapabilities(webdriver.Capabilities.firefox()).
   build();
 
+var testRunner = require('./test-runner');
+var sequentialForm = require('./sequential-form-test.js')(driver, By);
```

Comment:
Why passing `By`? Why not just require it there since we don't even use it in this file?